google/cloud/spanner_v1/database.py

```diff
@@ -733,11 +733,15 @@ def batch(self, request_options=None):
                 (Optional) Common options for the commit request.
                 If a dict is provided, it must be of the same form as the protobuf
                 message :class:`~google.cloud.spanner_v1.types.RequestOptions`.
+        :type max_commit_delay: :class:`datetime.timedelta`
```

Comment:
Is there any range of values that are valid for this option? If yes can we mention that range here for customers to understand the right values to pass.
When I look at autogenerated code it mentions the range as `0 and 500 ms`.
https://github.com/googleapis/python-spanner/blob/5b94dac507cebde2025d412da0a82373afdbdaf5/google/cloud/spanner_v1/types/spanner.py#L1445

Reply:
Done.